src/libopensc/card-mcrd.c

```diff
@@ -347,15 +347,18 @@ static int mcrd_init(sc_card_t * card)
 			if(apdu.sw1 != 0x90 && apdu.sw2 != 0x00)
 			{
 				sc_format_apdu(card, &apdu, SC_APDU_CASE_3, 0xA4, 0x04, 0x00);
-	                        apdu.lc = sizeof(EstEID_v35_AID);
-        	                apdu.data = EstEID_v35_AID;
-                	        apdu.datalen = sizeof(EstEID_v35_AID);
-                        	apdu.resplen = 0;
-	                        apdu.le = 0;
+							apdu.lc = sizeof(EstEID_v35_AID);
+							apdu.data = EstEID_v35_AID;
+							apdu.datalen = sizeof(EstEID_v35_AID);
+							apdu.resplen = 0;
+							apdu.le = 0;
 				r = sc_transmit_apdu(card, &apdu);
-	                        SC_TEST_RET(card->ctx, SC_LOG_DEBUG_NORMAL, r, "APDU transmit failed");
-        	                sc_debug(card->ctx, SC_LOG_DEBUG_VERBOSE, "SELECT AID: %02X%02X", apdu.sw1, apdu.sw2);
-				if (apdu.sw1 != 0x90 && apdu.sw2 != 0x00) {
+							SC_TEST_RET(card->ctx, SC_LOG_DEBUG_NORMAL, r, "APDU transmit failed");
```

Comment:
indenting

Reply:
Same here, spaces and tabs mixed before and do I leave as it is?